### PR TITLE
Clarified wording in tutorial text.

### PR DESCRIPTION
**Pull request by burhanloey**
I'm not a native speaker but the following sentence doesn't sound right. I checked with [this site](http://www.reverso.net/spell-checker/english-spelling-grammar/) and made a minor correction. CMIIW.

### CODE REVIEW

File: docs/intro/tutorial05.txt [hunk 1]
```diff
@@ -188,9 +188,9 @@ Put the following in the ``tests.py`` file in the ``polls`` application:
             future_question = Question(pub_date=time)
             self.assertIs(future_question.was_published_recently(), False)
 
-What we have done here is created a :class:`django.test.TestCase` subclass
-with a method that creates a ``Question`` instance with a ``pub_date`` in the
-future. We then check the output of ``was_published_recently()`` - which
+What we have done here is that we created a :class:`django.test.TestCase`
```
[the remark this answers is not in the document]
Comment (burhanloey) [reply docs/intro/tutorial05.txt]: Much better. Less is more. I'll make the change.

**Comment by carltongibson** [comment]
Hi @burhanloey. 

I've squashed the commits and adjusted the commit message to the [suggested format](https://docs.djangoproject.com/en/dev/internals/contributing/committing-code/#committing-guidelines). 

Thanks for the input!

**Comment by burhanloey** [comment]
You're welcome.

Woops, my bad not following guidelines. Thank you for the adjustment.

**Comment by timgraham** [comment]
I'm okay with the changes, although generally I try to avoid trivial wording changes unless it's a big problem as it creates extra work for documentation translators. I guess "simplified" would be the correct term, rather than "clarified".

**Pull request merged**